test/federated/test_federated_client.py

```diff
@@ -11,43 +11,51 @@
 PRINT_IN_UNITTESTS = False
 
 # To make execution deterministic to some extent
-# for more information - refer https://pytorch.org/docs/stable/notes/randomness.html
+# For more information - refer https://pytorch.org/docs/stable/notes/randomness.html
 torch.manual_seed(0)
 torch.cuda.manual_seed_all(0)
 torch.backends.cudnn.benchmark = False
 torch.backends.cudnn.deterministic = True
 
 
 def test_add_dataset():
+    # Create a client to execute federated learning
     fed_client = FederatedClient()
-
+    # Create a dataset
     dataset = "my_dataset"
-    fed_client.add_dataset(dataset, "string_dataset")
+    key = "string_dataset"
+    # add new dataset
```

Comment:
Could you use 'A' in place of 'a' such that it is the same as above (with big "C")?